app/views/feeds/_form_main.html.erb

```diff
@@ -78,37 +78,44 @@
   <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.episode_offset_seconds") %>"><span class="material-icons">help</span></button>
 </div>
 
-<div class="col-12 mb-4">
+<div class="col-12 mb-4" data-controller="toggle-field">
   <hr>
   <h3><%= t(".headers.audio_format") %></h3>
   <p><%= t(".help.audio_format") %></p>
 
   <div class="row my-4">
     <div class="col-3">
       <div class="form-floating">
-        <%= form.select :audio_format, audio_format_options %>
-        <%= form.label :audio_format %>
+        <%= form.select :audio_type, audio_format_options, {include_blank: true}, data: {action: "toggle-field#displayOnlyThisField"} %>
+        <%= form.label :audio_type %>
       </div>
     </div>
 
-    <div class="col-3">
+    <div class="col-3 <%= display_bitrate(feed) %>" data-toggle-field-target="field" field="mp3">
```

Comment:
for this and the below fields, 2 things are defined on this div line: initial display class, and display toggle.

the initial display is in `feeds_helper` and is based on `audio_format[:f]` or `audio_format.blank?` depending on the field.

the toggle is triggered by the change in value of `audio_type` (aka `audio_format[:f]`). so this one displays only with `mp3`, bitdepth with either `wav` or `flac`, and the other two with any of the three types. and `d-none` for blank.